### ISSUE TITLE
Support Linux brew

### ISSUE THREAD

**Issue by itaysk** [aquasecurity/trivy#205]
TL;DR: we are using an old version of goreleaser, and should update. This brings support for Linux brew as well.

- I tried to install Trivy using brew on Linux, and it didn't work. The binaries were for macOS. 
- Goreleaser supports linux brew since `v0.110.0`. Trivy's CI is using a [custom docker image](https://github.com/aquasecurity/trivy/blob/84fec5ae12e215e349974d968d01d96b4f93657c/ci/Dockerfile) which is based from [bepsays/ci-goreleaser](https://hub.docker.com/r/bepsays/ci-goreleaser)
- this image is currently [using `v0.95.0` of goreleaser](https://github.com/bep/dockerfiles/blob/fdd3638cd8d028293a637ca7d967df298343a4e5/ci-goreleaser/Dockerfile#L5)
- the [latest one they have is `v0.109.0`](https://github.com/bep/dockerfiles/blob/ce99f1dc6b621ede100f6b8e7340a30e44630345/ci-goreleaser/Dockerfile#L5)
- (the latest version of goreleaser os is `v1.119.0`). 

My plan is to refactor the CI docker image, and perhaps the CI pipeline to support newer goreleaser.
